compiler/cpp/src/thrift/generate/t_erl_generator.cc

```diff
@@ -984,10 +988,13 @@ void t_erl_generator::export_types_string(string name, int num) {
 }
 
 void t_erl_generator::export_function(t_function* tfunction, string prefix) {
-
+  t_struct::members_type::size_type num = tfunction->get_arglist()->get_members().size();
+  if (num > std::numeric_limits<int>().max()) {
```

Comment:
I'm getting a warning at ` if (num > std::numeric_limits<int>().max()) { ` in this file and also in the erlang generator (2 occurences):

`C4018: ">": Conflict between "signed" and "unsigned"`